server/controllers/calls.js

```diff
@@ -25,13 +57,23 @@ function outcomeIsValid(outcome) {
   return validOutcomes.includes(outcome.toUpperCase());
 }
 
-function updateNoCallContact(outcome, id) {
-  const { updateContactDoNotCallById, updateContactInvalidNumberById } = contactsService;
-  const outcomeMap = {
-    DO_NOT_CALL: updateContactDoNotCallById,
-    BAD_NUMBER: updateContactInvalidNumberById
+function validateStatus(currStatus, prevStatus) {
```

Comment:
Perhaps rename this to `validateStatusForUpdate` to be more specific that other statuses are valid, just that we don't support directly updating to them?

Reply:
✔️